starknet_py/tests/e2e/client/full_node_test.py

```diff
@@ -2,21 +2,34 @@
 
 import pytest
 
+from starknet_py.common import create_casm_class
 from starknet_py.contract import Contract
+from starknet_py.hash.address import compute_address
+from starknet_py.hash.casm_class_hash import compute_casm_class_hash
 from starknet_py.hash.selector import get_selector_from_name
 from starknet_py.hash.storage import get_storage_var_address
 from starknet_py.net.account.account import Account
 from starknet_py.net.client_errors import ClientError
 from starknet_py.net.client_models import (
     BlockHashAndNumber,
+    Call,
     ContractClass,
     DeclareTransaction,
+    DeclareTransactionTrace,
+    DeployAccountTransactionTrace,
+    InvokeTransactionTrace,
     SierraContractClass,
+    SimulatedTransaction,
     SyncStatus,
     TransactionType,
 )
 from starknet_py.net.full_node_client import _to_rpc_felt
 from starknet_py.net.models import StarknetChainId
+from starknet_py.tests.e2e.fixtures.constants import (
+    CONTRACTS_COMPILED_DIR,
+    CONTRACTS_COMPILED_V1_DIR,
+)
```

Comment:
I know it was there before, but maybe change this to `CONTRACTS_COMPILED_V0_DIR`.